index.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
```

Comment:
This file is kind of "test runner" so maybe we could rename it? OTOH `index.js` in `tests` directory sounds like a valid entry point :thinking: